lib/UserBackend.php

```diff
@@ -328,7 +331,7 @@ public function getDisplayName($uid) {
 			}
 		}
 
-		return false;
+		return $uid;
```

Comment:
did you check internal usage of this method? are we somewhere expecting or acting specifically to `false`?

Reply:
All other user backends implement it that way. The interface actually requires a string type and doesn't allow boolean values anymore

Reply:
No doubt about this, my point was to double check we don't have odd internal behavior that relies on the false.

Reply:
… but that is not the case :heavy_check_mark: